src/codecs/json.js

```diff
@@ -2,7 +2,7 @@
 
 import { codec } from './codec.js'
 
-export default codec({
+export const jsonCodec = codec({
```

Comment:
As per comment above

```suggestion
export const json = codec({
```